src/stories/LeaderCard.stories.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+import moment from 'moment';
+
+import { Text } from 'components';
+import { LeaderCard } from 'explorer-components';
+
+storiesOf('LeaderCard', module).add('LeaderCard', () => (
```

Comment:
:) nice. However, we are not including these in storyboard. Just the general components (this is a specific component)

Reply:
sorry I meant to remove this, it was just for me to build quickly 🤒